### ISSUE TITLE
Add ability to stop & start database instances

### ISSUE THREAD

**Issue by M-Schiefer** [GoogleCloudPlatform/elcarro-oracle-operator#273]
**TL;DR**

Ability to scale down the database instance when not in use to save compute resources and scale it back up when needed. Ideally automatically supported by helm upgrade.

**Is your feature request related to a problem? Please describe.**

We have an important use-case, where in a scalable infrastructure environment (e.g. GKE), we want to scale up/down all resources of an application when they don't need to run, in order to save expensive compute resources.

The replica count of the statefulset of the database instance is not tracked in the instance CRD. A "helm upgrade" after a scale-down, to restore all statefulsets/deployments to the replicas defined in the helm chart thus works for all parts of our application, except for El Carro, which remains scaled down to 0 replicas. Scaling the statefulset manually is an option, but also not fully supported by the operator, so it may have negative side-effects.

**Describe the solution you'd like**
A flag to instruct the El Carro operator to cleanly stop/start the database instance (+ scale the statefulset down/up). If previously set to "scale down" to scale down, a "helm upgrade" should set it back to "scale up" and start the database instance pod again.

The resulting flow might look like:
1.	[CREATE] “helm upgrade --install” -> Instance CRD is deployed with “instance.Spec.IsStopped: false”
2.	[STOP] Set “instance.Spec.IsStopped: true” to stop the DB
3.	[START] “helm upgrade –install” -> Sets “instance.Spec.IsStopped: false” (because that’s the only difference between the CRD in k8s and in the chart), triggering startup of the DB

Thank you for considering this request!

Kind regards,
Maximilian Schiefer (for Regnology)
